packages/react-components/react-theme/.babelrc.json

```diff
@@ -1,4 +1,3 @@
 {
-  "presets": [],
```

Comment:
looks like we missed few packages. packages without mapping trigger babelpostprocess errors

<img width="1759" alt="image" src="https://user-images.githubusercontent.com/1223799/227182123-5f9cca03-357f-40a3-a648-844f7989a824.png">


We need to use that aliasing mapping for every package that uses swc + babelpostprocess to point babel always to ESM